library/core/src/slice/mod.rs

```diff
@@ -5390,7 +5390,7 @@ unsafe impl<'a, T> TrustedRandomAccess for ChunksMut<'a, T> {
 /// This struct is created by the [`chunks_exact`] method on [slices].
 ///
 /// [`chunks_exact`]: ../../std/primitive.slice.html#method.chunks_exact
-/// [`remainder`]: ../../std/slice/struct.ChunksExact.html#method.remainder
+/// [`remainder`]: crate::slice::ChunksExact::remainder
```

Comment:
This is in the `slice` module, so `ChunksExact` is already in scope.
```suggestion
/// [`remainder`]: ChunksExact::remainder
```